src/module/system/text-editor.ts

```diff
@@ -695,6 +696,7 @@ class TextEditorPF2e extends TextEditor {
                 pf2Label: localize("DCWithName", { name }),
                 pf2Adjustment: Number(params.adjustment) || null,
                 pf2Roller: params.roller || null,
+                pf2TargetSelf: params.targetSelf,
```

Comment:
no `pf2`, and make it `targetOwner` so that there's no confusion of perspective.

Reply:
targetOwner is a much better name! Thanks!